docs/Applications.md

```diff
@@ -111,6 +112,30 @@ contxtSdk.coordinator.applications
   .then((featuredApplications) => console.log(featuredApplications))
   .catch((err) => console.log(err));
 ```
+<a name="Applications+getGroupings"></a>
+
+### contxtSdk.coordinator.applications.getGroupings(applicationId) ⇒ <code>Promise</code>
+Gets the application groupings (and application modules) of an application
+that are available to the currently authenticated user.
+
+API Endpoint: '/applications/:applicationId/groupings'
+Method: GET
+
+**Kind**: instance method of [<code>Applications</code>](#Applications)  
+**Fulfill**: <code>ContxtApplicationGrouping[]</code>  
+**Reject**: <code>Error</code>  
+
+| Param | Type |
+| --- | --- |
+| applicationId | <code>string</code> | 
+
+**Example**  
+```js
+contxtSdk.coordinator.applications
+  .getGroupings('31b52932-e25c-42d6-9c60-2f82d38d1e9c')
```

Comment:
Not that it _REALLY_ matters, but I believe the application IDs are just integers and not UUIDs, so it might be a little more clear if it was just `.getGroupings(1)`